### ISSUE TITLE
Dialog is not resizable?

### ISSUE THREAD

**Issue by hoseyjoe** [haxeui/haxeui-core#14]
Setting the width/height doesnt appear to affect the window

## Current Behavior
If you change the height then the window appears taller but the buttons remain at launch position
No effect if you cahnge the width.
## Test app / minimal test case
Where accountView is a VBox
```
var dOptions:DialogOptions = new DialogOptions();
dOptions.title = "Edit Account";
dOptions.addStandardButton(DialogButton.CLOSE);
dOptions.addStandardButton(DialogButton.CANCEL);
var d:Dialog = mainView.screen.showDialog(accountView, dOptions);

d.height = 500;
d.width = 500;
```
## Your Environment
NME and HTML tested

**Comment by ianharrigan** [comment]
What would you say is the expected behaviour? It should recenter if the app resizes and its a fixed width? And change size of its using percent sizes?

**Comment by hoseyjoe** [comment]
After reading your response I think their are 2 types of windows, whether
or not they are both considered "Dialogs" I dont know.
1. A fixed width dialog. I guess the expected behavior was that I could
give it the size I want and call it good. I think Dialogs work in this
controlled layout environment, meaning, it looks the same irregardless of
screen size. Question and answer windows like: Are you sure you want cheese
with that wine?  Yes No
Recenter on app resize would be great.

2. A working dialog. In this case you would generally want as much screen
real estate that is feasible; a half inch border for example.  A pop up
image editor or something
Recenter/resize on app resize would be great

**Comment by ianharrigan** [comment]
Yeah, i agree. I think that all dialogs, should recenter on app resize, and if they are using a percent width and/or height they should also resize based on that screen size before the recenter.
